source/js/components/petition/donation-modal.jsx

```diff
@@ -58,8 +69,8 @@ class DonationModal extends Component {
   render() {
     return (
       <div ref={(e) => (this.domLocation = e)}>
-        <div className="modal-underlay" ref={(e) => (this.fragment = e)}>
-          <div className="modal show" role="dialog">
+        <div className={MODAL_OVERLAY_CLASS} ref={(e) => (this.fragment = e)}>
+          <div className={classNames(`modal`, MODAL_CLASS)} role="dialog">
```

Comment:
We need to keep `modal` class here as we are still relying on Bootstrap's stylings for `.modal`